### DISCUSSION TITLE
Fetures and suggestions

### DISCUSSION THREAD

**Discussion by ghost** [akanshSirohi/ShareX#57] [Ideas]
Dear Akansh!
I've used your app for an hour and I have a few feature suggestions that may be of interest to you because they potentially can improve user experience.

1. **BUG**: App froze during first launch when battery/optimization permission was called. Force closed the app then launched it and proceeded as usual giving the permission.
2. **FEATURE**: I was not able to choose "Download" folder in "Define root of ShareX" section. This problem can be solved if you add an option to run the app as root.
3. **FEATURE**: Widget with Start/Stop/QR code/Share IP and actual IP:port would be very nice.
4. **FEATURE**: Always visible (not hidden in the menu) buttons for: New folder / Download / Rename / Delete / Move? / Copy?
5. **FEATURE**: Drag and Drop for folders (you drag the folder and it uploads the whole folder)
6. **FEATURE**: Drag one file / multiple marked files to the folder
7. **FEATURE**: View options for the content: Grid / List / Columns / Miniatures with image preview
8. **FEATURE**: Add additional info for the files/folders: file type / date & time of change

Thank you for your time and effort to bring this project to life. I hope my suggestions will be somewhat useful to you.
I'll make sure to spread the word around about your app and to buy you a coffee. :)

**Comment by akanshSirohi** [comment]
@iamnytskyi Thank you so much for taking out your time to review my app and supporting the app! I will try to fix this bug you mentioned as soon as possible and introduce more features including some of your suggestions in the future.

**Reply by ghost** [reply]
Dear Akansh!
Where can I submit ukrainian and russian translations for your app?
Is there a crowdin project maybe?

**Reply by akanshSirohi** [reply]
There is no any crowdin project for now, but you can share me that by any other means, contact me on twitter, we can discuss it there.
Twitter: @akansh__sirohi

**Reply by ghost** [reply]
[ShareX Translation to UA and RU.xlsx](https://github.com/akanshSirohi/ShareX/files/12259649/ShareX.Translation.to.UA.and.RU.xlsx)

**Comment by akanshSirohi** [comment]
@iamnytskyi Thank you for your support. I will make every effort to promptly implement the suggested changes. However, my current schedule is quite demanding, leaving me with limited time. Nonetheless, I assure you that I will diligently work on updating the application as soon as I find a suitable window of time.
